utensor_cgen/core.py

```diff
@@ -46,6 +47,12 @@ def generate(self, src_fname: str):
     print("Parsing {}".format(self.pb_file))
     graph_info, layers = parse_pb(self.pb_file)
 
+    ref_table = get_refc_table(graph_info)
```

Comment:
Maybe leave the ref counting to a optimizer is a better design?
I'd like to talk to you about the interface of an optimizer.